pyrobosim/pyrobosim/data/test_world.yaml

```diff
@@ -74,15 +74,15 @@ rooms:
           x: 0.75
           y: 0.5
     wall_width: 0.2
-    color: [1, 0, 0]
+    color: "red"
 
   - name: bedroom
     footprint:
       type: box
       dims: [1.75, 1.5]
       offset: [2.625, 3.25]
     wall_width: 0.2
-    color: [0, 0.6, 0]
+    color: "#9ACD32" #yellowgreen
```

Comment:
same here re: using the same green as before, but in hex version.

Also, can you make the same modifications to `test_world_multirobot.yaml` which uses the same world?

Reply:
Done.